src/examples/GradientStroke.cpp

```diff
@@ -26,23 +26,24 @@ void tvgDrawCmds(tvg::Canvas* canvas)
 
     // linear gradient stroke + linear gradient fill
     auto shape1 = tvg::Shape::gen();
-    shape1->moveTo(100, 50);
-    shape1->lineTo(150, 50);
-    shape1->lineTo(150, 100);
-    shape1->lineTo(250, 100);
+    shape1->moveTo(150, 100);
+    shape1->lineTo(200, 100);
     shape1->lineTo(200, 150);
-    shape1->lineTo(150, 150);
+    shape1->lineTo(300, 150);
+    shape1->lineTo(250, 200);
+    shape1->lineTo(200, 200);
+    shape1->lineTo(200, 250);
+    shape1->lineTo(150, 300);
     shape1->lineTo(150, 200);
-    shape1->lineTo(100, 250);
+    shape1->lineTo(100, 200);
     shape1->lineTo(100, 150);
-    shape1->lineTo(50, 150);
-    shape1->lineTo(50, 100);
     shape1->close();
 
     shape1->stroke(0, 255, 0, 255);
     shape1->stroke(20);
     shape1->stroke(tvg::StrokeJoin::Miter);
     shape1->stroke(tvg::StrokeCap::Butt);
+
     auto fillStroke1 = tvg::LinearGradient::gen();
     fillStroke1->linear(50, 50, 200, 200);
```

Comment:
I'd shift the fill, so all 3 colors are visible, but as you wish
fillStroke1->linear(100, 100, 250, 250);

Reply:
the same in line 53

Reply:
Updated the code, do you satisfied with that? :)

Reply:
that is the result that we get now:
![ex_bad](https://user-images.githubusercontent.com/67589014/111772802-91193480-88ad-11eb-87c9-17746987b9f7.PNG)


The idea was to check whether a stroke and a fill have the same grad:
![ex_ok](https://user-images.githubusercontent.com/67589014/111772784-8c548080-88ad-11eb-8593-f4382f1226a9.PNG)

You changed the fill differently than I wanted, but it's up to you. In any case, the fill and the stroke's fill should be the same. That's why I wrote that the change also applies to line 53.